HelpSource/Classes/Collection.schelp

```diff
@@ -304,10 +304,53 @@ List[1, 2, 3, 3, 4, 3, 4, 3].occurrencesOf(3);
 ::
 
 method::sum
-Answer the sum of the results of strong::function:: evaluated for each item in the receiver. The function is passed two arguments, the item and an integer index.
+Answer the sum of all items or the results of the optional strong::function:: argument evaluated for each item in the receiver. The function is passed two arguments, the item and an integer index.
 code::
 List[1, 2, 3, 4].sum;
+
 (0..8).sum { |i| 1 / (2 ** i) };
+// Only item is used as index is not used in the function.
+::
+discussion::
+If there is no strong::function:: argument, it can be expressed mathematically as follows:
```

Comment:
The standard case in mathematics is that there *is* a function argument.
So you can write yours as \sum_{i = 0}^{8} \frac{1}{2^i}

You can add sclang examples with functions, for mean and product as well.